packages/database/src/server/TupaiaDatabase.js

```diff
@@ -1,20 +1,30 @@
 import winston from 'winston';
 import { types as pgTypes } from 'pg';
 
+import { Multilock } from '@tupaia/utils';
+
 import { BaseDatabase } from '../core';
 import { getConnectionConfig } from './getConnectionConfig';
 import { DatabaseChangeChannel } from './DatabaseChangeChannel';
 
 export class TupaiaDatabase extends BaseDatabase {
   static IS_CHANGE_HANDLER_SUPPORTED = true;
 
+  /**
+   * @privateRemarks
+   * No math here, just hand-tuned to be as low as possible while keeping all the tests passing.
+   */
+  static handlerDebounceDurationMs = 250;
```

Comment:
Now that this is only ever used by `TupaiaDatabase`, we can make this private:

```suggestion
  static #handlerDebounceDurationMs = 250;
```

(Previously I thought I needed `TupaiaDatabase` to inherit it from `BaseDatabase`)